### PR TITLE
Fixes an "OBJECT_STORE_HOSTNAME: unbound variable" error that occurs when restoring from a backup.

### PR DESCRIPTION
**Pull request by jala-dx**

### CODE REVIEW

**Comment by laverya** [comment]
close https://github.com/replicatedhq/kURL/pull/2678 if this is merged

File: addons/registry/2.7.1/tmpl-configmap-velero.yaml [hunk 1]
```diff
@@ -10,6 +10,9 @@ data:
     #!/bin/sh
     set -euo pipefail
     echo 'backup starting ...'
+    if [ -z "\$OBJECT_STORE_HOSTNAME" ]; then
+      export OBJECT_STORE_HOSTNAME="\$OBJECT_STORE_CLUSTER_IP"
```
Comment (laverya) [inline addons/registry/2.7.1/tmpl-configmap-velero.yaml:14]:
```suggestion
      export OBJECT_STORE_HOSTNAME=\$OBJECT_STORE_CLUSTER_IP # included for backwards compatibility for snapshot restores with pre-ipv6 snapshot restores
```

Comment (laverya) [reply addons/registry/2.7.1/tmpl-configmap-velero.yaml:14]: just always good to have a comment

**Review by laverya — DISMISSED** [review]

**Review by laverya — APPROVED** [review]

File: addons/registry/2.7.1/tmpl-configmap-velero.yaml [hunk 2]
```diff
@@ -10,6 +10,9 @@ data:
     #!/bin/sh
     set -euo pipefail
     echo 'backup starting ...'
+    if [ -z "\$OBJECT_STORE_HOSTNAME" ]; then
```
Comment (Sharpie) [inline addons/registry/2.7.1/tmpl-configmap-velero.yaml:13]: This test will abort the entire script when combined with `set -u`. An empty string _must be substituted_ to satisfy the undefined variable check:

```sh
if [ -z "\${OBJECT_STORE_HOSTNAME-}" ]; then
  export OBJECT_STORE_HOSTNAME=\$OBJECT_STORE_CLUSTER_IP
fi
```

See http://mywiki.wooledge.org/BashFAQ/112 for more details.

**Pull request merged**